src/org/traccar/reports/Summary.java

```diff
@@ -60,7 +63,14 @@ private static SummaryReport calculateSummaryResult(long deviceId, Date from, Da
                 speedSum += position.getSpeed();
                 result.setMaxSpeed(position.getSpeed());
             }
-            result.setDistance(ReportUtils.calculateDistance(firstPosition, previousPosition));
+            boolean ignoreOdometer = false;
+            if (device.getAttributes().containsKey(ReportUtils.IGNORE_ODOMETER)) {
```

Comment:
It would be nice to move this functionality somewhere in common place because it would be useful in multiple places. For example, I'm already using something similar for GT06 protocol encoder.